From 239743e0a0a675cc0f13caaed056b981d8ce2d08 Mon Sep 17 00:00:00 2001
From: Matthew Gerlach <matthew.gerlach@linux.intel.com>
Date: Mon, 2 May 2022 14:04:29 -0700
Subject: [PATCH 5/5] n6000: update device tree for 2GB of DDR

The Hard Processor System (HPS) on the n6000 has 2GB of DDR
connected.  Update the devicetree with the correct memory size.

Signed-off-by: Matthew Gerlach <matthew.gerlach@linux.intel.com>
---
 arch/arm/dts/socfpga_agilex_n6000-u-boot.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/dts/socfpga_agilex_n6000-u-boot.dtsi b/arch/arm/dts/socfpga_agilex_n6000-u-boot.dtsi
index 7dcdffcdaa..7092c2b1bc 100644
--- a/arch/arm/dts/socfpga_agilex_n6000-u-boot.dtsi
+++ b/arch/arm/dts/socfpga_agilex_n6000-u-boot.dtsi
@@ -14,8 +14,8 @@
 	};
 
 	memory {
-		/* 1GB */
-		reg = <0 0x00000000 0 0x40000000>;
+		/* 2GB */
+		reg = <0 0x00000000 0 0x80000000>;
 	};
 
 	chosen {
-- 
2.25.1

